src/display/api.js

```diff
@@ -755,22 +757,8 @@ class PDFDocumentProxy {
    */
 
   /**
-   * @returns {Promise<Array<OutlineNode>>} A promise that is resolved with
-   * an {Array} that is a tree outline (if it has one) of the PDF. The tree is
-   * in the format of:
-   * [
-   *   {
-   *     title: string,
-   *     bold: boolean,
-   *     italic: boolean,
-   *     color: rgb Uint8ClampedArray,
-   *     count: integer or undefined,
```

Comment:
This property isn't even listed in the `typedef`, strangely enough.

Reply:
I checked the code and the unit test and found two more properties that were missing. It should be complete and correct now.